### PR TITLE
Help people understand generated files are *theirs*

### PR DESCRIPTION
**Pull request by nicolas-grekas**
image

A response to https://twitter.com/fabpot/status/938074790023139329

### CODE REVIEW

File: src/Flex.php [hunk 1]
```diff
@@ -231,6 +232,15 @@ function ($value) {
             }
         }
 
+        if (null !== $manifest) {
+            array_unshift(
+                $this->postInstallOutput,
+                '',
+                '<info>A few files may have been created to configure newly installed packages.</>',
+                'Don\'t hesitate to <comment>review</>, <comment>edit</> and <comment>commit</> them: these files are <comment>*yours*</>.'
```
Comment (stof) [inline src/Flex.php]: I don't think you should put stars in it. If you want to make it bold, make it bold for real (I think the Console component supports it)

Comment (Pierstoval) [reply src/Flex.php]: The console supports it, but not all terminals 😉

Comment (nicolas-grekas) [reply src/Flex.php]: stars removed (bold+yellow didn't look nice)

File: src/Flex.php [hunk 2]
```diff
@@ -231,6 +232,15 @@ function ($value) {
             }
         }
 
+        if (null !== $manifest) {
+            array_unshift(
+                $this->postInstallOutput,
+                '',
+                '<info>A few files may have been created to configure newly installed packages.</>',
```
Comment (fabpot) [inline src/Flex.php]: Some files may have been created or updated to configure your newly installed packages.

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**